### PR TITLE
Homewokr-5

### PR DESCRIPTION
**Pull request by kostyaaderiho**

### CODE REVIEW

**Review by mikalai-sokarau — APPROVED** [review]
Hi, there is a requirement in the task to log `onunhandledrejection` errors, as soon as you add it the final mark will be 5.
Other things work as expected.
Thank you in advance :)

**Comment by kostyaaderiho** [comment]
> Hi, there is a requirement in the task to log `onunhandledrejection` errors, as soon as you add it the final mark will be 5.
> Other things work as expected.
> Thank you in advance :)

Actually, it's handled so if you throw an error in any controller like throw new Error('Error!'), it will be handled  in logErrors.ts file and save error log in info.log

File: source/homework-2/constants/messages.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const messages = {
+    404: 'Not found',
+    500: 'Internal Server Error'
+};
```
[the remark this answers is not in the document]
Comment (kostyaaderiho) [reply source/homework-2/constants/messages.ts:4]: sry I didn't get what you mean

File: source/homework-2/decorators/timing.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { logger } from '../utils/logger';
+
+export const timing = (fn: Function) => {
+    return async (...args: [Request, Response, NextFunction]) => {
+        const start = Date.now();
+
+        const result = await fn.apply(this, args);
```
[the remark this answers is not in the document]
Comment (kostyaaderiho) [reply source/homework-2/decorators/timing.ts]: okay I've updated

File: source/homework-2/constants/messages.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikalai-sokarau) [reply source/homework-2/constants/messages.ts:4]: I believe it's just a question of code style, but personally I prefer to use `enum` in such situations, this way:
```
export enum Messages {
    NotFound = 'Not found',
    InternalServerError = 'Internal Server Error'
}
```

**Comment by mikalai-sokarau** [comment]
> Actually, it's handled so if you throw an error in any controller like throw new Error('Error!'), it will be handled in logErrors.ts file and save error log in info.log


Indeed, missed it during the review, even better in this case! :)

**Pull request merged**